components/mdx/Heading.js

```diff
@@ -0,0 +1,27 @@
+const Heading = ({ children, id, fontSize = "medium", customClasses }) => {
```

Comment:
Added more details in the conversation with Eddie, but this component isn't needed as it's duplicating native markdown functionality.

Reply:
I have removed this component